### ISSUE TITLE
Table view cell drawing not updated correctly.

### ISSUE THREAD

**Issue by brutella** [twitter-archive/twui#123]
In the past I could draw a text inside a table view cell with a different color, depending on the cell state (selected or not) by using a text renderer object.

With the current master branch, this is not possible anymore, since the table view cell drawing is not updated correctly.

**Example**
I've updated the Example project to reproduce the problem [here](https://github.com/brutella/twui/commit/d2595826092d2284d089b893628143d0dba2ba41). The text renderer should draw the text in white color if the cell is selected, otherwise the title should be black. The problem is that the title color gets not updated correctly - after deselecting a cell, the title is still white and after selecting the title is still black.

**Comment by avaidyam** [comment]
Yeah, I see this too. Except with status indicators. Not sure why though.
